### ISSUE TITLE
Enable Guest Checkout

### ISSUE THREAD

**Issue by mlwoodward** [woodwarddesign/GWG-Clothing#4]
Guest Checkout is not working.

**Comment by mlwoodward** [comment]
We have been in conversation with Derek about this issue.  The front end is configured correctly.  There is a setting on the backend that needs adjusting.  Once we have access to the site, we will review the issue and give the client feedback.

**Comment by mlwoodward** [comment]
We will review the issue and give feedback.  The client is awaiting Derek's feedback.  He has been working on it and plans on deploying fix today.  This is a hack they did that disables the current solution in the system.

**Comment by gwgjan** [comment]
Latest feedback from Derek: 
We ran into a bug which slowed us down - the custom cart design for GWG basically makes it harder to reimplement guest checkout as we can't use existing Magento stock logic. Also another critical project popped up which has booked a majority of our senior resources. 

As I've discussed with you it's about bandwidth as well outside of bugs - I'm hoping we can deploy this evening.
